app/ts/components/pages/Home.tsx

```diff
@@ -132,11 +142,20 @@ function FirstCard(param: FirstCardParams) {
 						</div>
 						: <p style = 'color: var(--subtitle-text-color);' class = 'subtitle is-7'> { ` You can change active address by changing it directly from ${ getPrettySignerName(param.tabState?.signerName ?? 'NoSignerDetected') }` } </p>
 					}
-				</> : <div style = 'display: flex; justify-content: space-between; padding-top: 10px'>
-					<label class = 'form-control'>
+				</> : <div style = 'justify-content: space-between; padding-top: 10px'>
+					<label class = 'form-control' style = 'width: 120px;'>
```

Comment:
Hard coding widths for labels is risky because translations, fonts, accessibility features, etc. can all break it.  Is there any way to make this more responsive?